servicetalk-http-api/src/main/java/io/servicetalk/http/api/MultiAddressHttpClientBuilder.java

```diff
@@ -294,10 +100,16 @@
      *     filter1 ⇒ filter2 ⇒ filter3 ⇒ client
      * </pre>
      *
+     * @deprecated Use
+     *   {@link #initializer(SingleAddressInitializer)} and
+     *   {@link SingleAddressHttpClientBuilder#appendClientFilter(StreamingHttpClientFilterFactory)}
+     *   on the last argument of
+     *   {@link SingleAddressInitializer#initialize(String, Object, SingleAddressHttpClientBuilder)}.
      * @param factory {@link MultiAddressHttpClientFilterFactory} to decorate a {@link StreamingHttpClient} for the
      * purpose of filtering.
      * @return {@code this}
      */
+    @Deprecated
     public abstract MultiAddressHttpClientBuilder<U, R> appendClientFilter(
```

Comment:
can we remove `appendClientFilter*` methods from this PR? IIUC the intention is https://github.com/apple/servicetalk/pull/1722 will deprecate it, and can be back ported to 0.41 ... but this PR won't be back ported so we should be able to remove.

Reply:
That's true, thanks for keeping an eye on that.

Reply:
I removed the `MultiAddressHttpClientFilterFactory` and references to it in the latest commit. #1722 will be updated with relevant deprecations too.